Library/Homebrew/brew.sh

```diff
@@ -345,7 +345,13 @@ else
   HOMEBREW_OS_USER_AGENT_VERSION="$HOMEBREW_OS_VERSION"
 
   # Ensure the system Curl is a version that supports modern HTTPS certificates.
-  HOMEBREW_MINIMUM_CURL_VERSION="7.41.0"
+  if [[ -z $HOMEBREW_FORCE_HOMEBREW_ON_LINUX ]] && [[ -z $HOMEBREW_DEVELOPER ]] && [[ -z $HOMEBREW_ON_DEBIAN7 ]]
```

Comment:
1. Why do you split conditionals?
2. I'm a bit confused -- all of the conditional should be reversed (`-n`), no? Because you want to lower the minimum version on only when all of the vars are set.

```suggestion
  if [[ -n $HOMEBREW_FORCE_HOMEBREW_ON_LINUX && -n $HOMEBREW_DEVELOPER && -n $HOMEBREW_ON_DEBIAN7 ]]
```

Reply:
Oups I made a mistake, fixed.